src/routing/http_wrapper.h

```diff
@@ -9,7 +9,7 @@ Copyright (c) 2015-2024, Julien Coupey.
 All rights reserved (see LICENSE).
 
 */
-#include "../include/rapidjson/include/rapidjson/document.h"
+#include "<rapidjson/document.h>"
```

Comment:
couldn't this be worked around in vroom with a `make` flag to optionally add `-I ../include/rapidjson/include` and keep this instead, so people/package builders can use system packages if they want to?

Reply:
though not sure about rapidjson in particular which didn't release in a decade or so..